cmd/sovereignnode/incomingHeader/incomingEventsProcessor.go

```diff
@@ -65,9 +66,21 @@ func (iep *incomingEventsProcessor) processIncomingEvents(events []data.EventHan
 		case topicIDDeposit:
```

Comment:
I think that the topic IDs have to be renamed according to what they do:
topicIDDeposit -> something like incomingTransfer
topicIDExecuteBridgeOps -> incomingConfirmationsForOutgoingTransfers

Reply:
variable names updated